Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml.cs

```diff
@@ -224,6 +225,16 @@ public Visibility TabStripVisibility
         public static readonly DependencyProperty TabStripVisibilityProperty =
             DependencyProperty.Register("TabStripVisibility", typeof(Visibility), typeof(HorizontalMultitaskingControl), new PropertyMetadata(Visibility.Visible));
 
+        private bool firstLoad = true;
 
+        private void TabViewItem_Loaded(object sender, RoutedEventArgs e)
+        {
+            // fixes issue where tab would not show as selected when opened with path argument
+            if(firstLoad)
+            {
+                firstLoad = false;
+                (sender as TabViewItem).IsSelected = true;
```

Comment:
Would `(sender as TabViewItem).Loaded -= TabViewItem_Loaded` work instead of using the `firstLoad` thing?

Reply:
I don't think so, the event would have to be removed from all TabViewItems since the event is in the template.

Reply:
Sorry I had misunderstood your intentions here.